### ISSUE TITLE
PVD font setting

### ISSUE THREAD

**Issue by WWebber** [NVIDIAGameWorks/PhysX#175]
How can I change the font for the list-view?
There's always a black rectangle shown at the end..

![image](https://user-images.githubusercontent.com/20129358/64120638-0b05cd80-cd9d-11e9-83bd-b60520d95ae7.png)

**Comment by Pierre-Terdiman** [comment]
Looks like the black rectangle indicates the word has been cut because the column is too small. You should be able to resize the column and make it wider with the mouse.

**Comment by WWebber** [comment]
yes. Thats right, but I don't like the font anyway - it's too big. The good old MS-SansSerif is still the best for such displays..

Here, the better font is still in use:
[https://developer.nvidia.com/physx-visual-debugger](https://developer.nvidia.com/physx-visual-debugger)
